src/app/src/components/FacilityListItemsTableRow.jsx

```diff
@@ -1,27 +1,43 @@
-import React, { memo } from 'react';
+import React from 'react';
 import { bool, func, number, oneOfType, string } from 'prop-types';
 import { Link } from 'react-router-dom';
 import TableRow from '@material-ui/core/TableRow';
 import TableCell from '@material-ui/core/TableCell';
-import isEqual from 'lodash/isEqual';
+import Button from '@material-ui/core/Button';
+import cond from 'lodash/cond';
+import matches from 'lodash/matches';
+import conforms from 'lodash/conforms';
+import identity from 'lodash/identity';
+import isFunction from 'lodash/isFunction';
+import constant from 'lodash/constant';
+import stubTrue from 'lodash/stubTrue';
 
 import { facilityListItemStatusPropType } from '../util/propTypes';
 
 import { listTableCellStyles } from '../util/styles';
 
 import { makeFacilityDetailLink } from '../util/util';
 
-const propsAreEqual = (prevProps, nextProps) =>
-    isEqual(prevProps.rowIndex, nextProps.rowIndex)
-    && isEqual(prevProps.countryCode, nextProps.countryCode)
-    && isEqual(prevProps.name, nextProps.name)
-    && isEqual(prevProps.address, nextProps.address)
-    && isEqual(prevProps.status, nextProps.status)
-    && isEqual(prevProps.hover, nextProps.hover)
-    && isEqual(prevProps.newFacility, nextProps.newFacility)
-    && isEqual(prevProps.oarID, nextProps.oarID);
+const makeTableRowStyles = cond([
```

Comment:
I like the power that `cond` and `conforms` give us, but we should consider that in cases like this where we are mapping 2 values that don't have any complex interactions, a lower-order solution is more compact and arguably more easily understood.

```javascript
const makeTableRowStyles = ({ handleSelectRow, isRemoved}) => {
    const pointerStyle = isFunction(handleSelectRow) ? { cursor: 'pointer' } : {};
    const opacityStyle = isRemoved ? { opacity: '0.6' } : {};
    return merge(pointerStyle, opacityStyle);
}
```

Reply:
I changed this in 90eb513ee385aaec7cc9275ca5a12b9f6731323c to be:

```
const makeTableRowStyles = ({ handleSelectRow, isRemoved }) => ({
    cursor: isFunction(handleSelectRow) ? 'pointer' : 'auto',
    opacity: isRemoved ? '0.6' : '1.0',
});
```